projects/tenio/index.js

```diff
@@ -0,0 +1,17 @@
+const COLLATERAL_PER_TOKEN = 0.10;  // ETH por fragmento
+const CONTRACT = '0xF0076BFF99424EAC54271E278A73AB60A9f47dF1';  // dirección checksum final
```

Comment:
this contract does not exist: https://etherscan.io/address/0xF0076BFF99424EAC54271E278A73AB60A9f47dF1